tests/fsharp/Compiler/ErrorMessages/NameResolutionTests.fs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace FSharp.Compiler.UnitTests
+
+open NUnit.Framework
+open FSharp.Compiler.SourceCodeServices
+
+[<TestFixture>]
+module ``NameResolutionTests`` = 
+    
+    [<Test>]
+    let `` FieldNotInRecord `` () =
+        CompilerAssert.TypeCheckSingleError
+            """
+type A = { Hello:string; World:string }
+type B = { Size:int; Height:int }
+type C = { Wheels:int }
+type D = { Size:int; Height:int; Walls:int }
+type E = { Unknown:string }
+type F = { Wallis:int; Size:int; Height:int; }
+
+let r:F = { Size=3; Height=4; Wall=1 }
+            """
+            FSharpErrorSeverity.Error
+            1129
+            (9, 31, 9, 35)
+            "The record label 'Wall' is not defined."
+
+    [<Test>]
+    let `` RecordFieldProposal `` () =
+        CompilerAssert.TypeCheckSingleError
+            """
+type A = { Hello:string; World:string }
+type B = { Size:int; Height:int }
+type C = { Wheels:int }
+type D = { Size:int; Height:int; Walls:int }
+type E = { Unknown:string }
+type F = { Wallis:int; Size:int; Height:int; }
+            
+let r = { Size=3; Height=4; Wall=1 }
+            """
+            FSharpErrorSeverity.Error
+            39
+            (9, 29, 9, 33)
+            "The record label 'Wall' is not defined."
+
+    [<Test>]
+    let `` GlobalQualifierAfterDot `` () =
+        CompilerAssert.TypeCheckSingleError
+            """
+let x = global.System.String.Empty.global.System.String.Empty
+            """
```

Comment:
This test fails because the parsing of the test file fails due to the same error as the error to be tested ('global' may only be used as the first name in a qualified path):

![image](https://user-images.githubusercontent.com/21338071/61211132-fee78300-a6fe-11e9-922f-5f31c8761133.png)

Any guidance on this?

Reply:
I think we need to add a new assertion function that checks parse errors instead of type check errors

Reply:
Should i open a new issue for this? I could also have a go at implementing it, although i don't have too much experience with NUnit. For this PR, i guess it is then better to only migrate the two working tests for now and migrate the 'global' test after the new assert function is there?

Reply:
Yes that's a good idea. Let's get the easy ones into the bank and I'll try to add the new function